frontend/src/v4/routes/teamspaceSettings/teamspaceSettings.component.tsx

```diff
@@ -266,6 +277,67 @@ export class TeamspaceSettings extends PureComponent<IProps, IState> {
 		);
 	}
 
+	private renderPermissionLogOption = () => {
+		const { isTeamspaceAdmin } = this.props;
+		if (!isTeamspaceAdmin) {
+			return null;
+		}
+		return (
+			<PermissionsLogContainer gap="10px" container direction="column" wrap="nowrap">
```

Comment:
![image](https://github.com/user-attachments/assets/c31fd47f-b9ca-49a7-a11e-2aa15dad2002)


Sorry Can we change the wording on this please:

With the section title changed to `Audit log`
And the caption blurb to `Download the audit trail of all access-related actions performed within the specified date range`